src/pages/events/summit-event-list-page/components/GridDisplayControl/GridDisplayControl.jsx

```
@@ -0,0 +1,105 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * */
+
+import React, { useEffect, useMemo, useState } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(unused-imports/no-unused-imports)